### PR TITLE
feat(atomic): atomic-pager component completion

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-372
Screen Shot 2021-01-29 at 4 45 09 PM

Inspired from https://vapor.cloud.coveo.com/#/components/Pagination & https://docs.coveo.com/

### CODE REVIEW

File: packages/atomic/src/components/atomic-pager/atomic-pager.tsx [hunk 1]
```diff
@@ -29,52 +31,78 @@ export class AtomicPager implements InitializableComponent {
...
+    });
   }
 
-  private get backButton() {
-    if (!this.pagerState.hasPreviousPage) {
-      return null;
-    }
-
-    const icon = '<';
+  private buildButton(options: {
```
Comment (btaillon) [inline packages/atomic/src/components/atomic-pager/atomic-pager.tsx:59]: Buttons should probably include [`aria-current`](https://www.w3.org/TR/wai-aria-1.1/#aria-current), which seems to be recommended for pagination.

File: packages/atomic/src/components/atomic-pager/atomic-pager.tsx [hunk 2]
```diff
@@ -29,52 +31,78 @@ export class AtomicPager implements InitializableComponent {
...
-          aria-label="Previous page"
-          onClick={() => {
-            this.pager.previousPage();
-          }}
+          part={options.part}
+          class={`text-secondary ${
+            options.disabled ? 'opacity-50 cursor-not-allowed' : ''
+          }`}
+          disabled={options.disabled}
+          aria-hidden={options.disabled}
```
Comment (btaillon) [inline packages/atomic/src/components/atomic-pager/atomic-pager.tsx:74]: I'm not sure if I'd use `aria-hidden` here, given that the button is disabled anyways. I haven't tested, but I'm guessing setting `aria-hidden` to `true` on an element focused by accessibility tools may cause issues.

**Review by btaillon — APPROVED** [review]

**Comment by nrawji** [comment]
I think the page numbers would look better circular like on the docs website https://docs.coveo.com/en/search/#q=facet&t=All&sort=relevancy but that's just a matter of opinion I guess.

**Review by olamothe — APPROVED** [review]

**Comment by olamothe** [comment]
Might be an illusion, but it looks like the numbers are not well aligned within the center of the circle ? 🤔

**Review by samisayegh — APPROVED** [review]

**Pull request merged**